zerver/openapi/zulip.yaml

```diff
@@ -11897,8 +11899,9 @@ paths:
                           streams:
                             type: array
                             description: |
-                              An array of objects where each object contains details of unread
-                              messages of a single subscribed stream, including muted streams.
+                              An array of dictionaries where each dictionary contains details of all
+                              unread messages of a single subscribed stream. This includes muted streams
+                              and muted topics, even though those messages are excluded from count.
```

Comment:
```suggestion
                              and muted topics, even though those messages are excluded from `count`.
```

Reply:
That wasn't in time for the merge, so fixed in 9e3133e8200d408480cc9dce69cc0a418a93bf83.